PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Description
+
+<!-- Please include a summary of the changes and the related issue. Include any relevant context here. -->
+
+## Issue Reference
+
+<!-- Link to the related issue in the repository. -->
+Fixes #(issue)
+
+## PR Details
+
+### Type of Change
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Breaking change
+- [ ] Documentation update
+- [ ] Code refactor
+
+## Checklist
+
+Please **ensure** the following before submitting the PR:
+
+- [ ] I have **tested** the changes locally, and they work as expected.
+- [ ] The code follows the project's coding standards and conventions.
+- [ ] Any **query changes** have been verified and are working correctly.
+- [ ] I have **updated relevant documentation** (if needed).
+- [ ] I have added unit **tests** or performed manual testing where necessary.
+- [ ] I have **mentioned** the person/team responsible for reviewing this PR. 
```

Comment:
This check box can go too, we don't need the contributor to know who should review.